Tropos/Sources/Controllers/CourierClient.swift

```diff
@@ -1,8 +1,21 @@
 import Courier
 
+@objc protocol TimeZone {
+    var secondsFromGMT: Int { get }
+}
+extension NSTimeZone: TimeZone {}
+
 @objc(TRCourierClient) final class CourierClient: NSObject {
     private let instance: Courier
 
+    class func channelNameForTimeZone(timeZone: TimeZone) -> String {
+        let seconds = timeZone.secondsFromGMT
+        let sign = seconds < 0 ? "minus" : "plus"
+        let hours = abs(seconds) / 3600
+        let minutes = (abs(seconds) % 3600) / 60
+        return String(format: "%@ %02d%02d", sign, hours, minutes)
```

Comment:
Thanks for explaining! Then I guess it doesn't really matter. The reason I noticed is because I also read the server-side implementation and was expecting there to be a dash.